DataFormats/L1TParticleFlow/interface/PFTau.h

```diff
@@ -41,21 +41,40 @@ namespace l1t {
     float chargedIso() const { return iso_; }
     float fullIso() const { return fullIso_; }
     int id() const { return id_; }
+
+    void setZ0(float z0) { setVertex(reco::Particle::Point(0, 0, z0)); }
+    void setDxy(float dxy) { dxy_ = dxy; }
+
+    float z0() const { return vz(); }
+    float dxy() const { return dxy_; }
+
+    bool passMass() const { return (mass() < 2 + pt() / 40); }
```

Comment:
@violatingcp ?   I have no more insight. 
Regarding `40` vs `40.` since pt() returns a float I don't think it matters but if we want to be really correct it should be `40.f`

Reply:
Sure, does not matter (seems to me it retraces back to [this](https://cmssdt.cern.ch/dxr/CMSSW/source/DataFormats/Candidate/interface/LeafCandidate.h#146) through `L1Candidate`)). Just the meaning was obscure to me.

Reply:
Defined the 40.0 as a constant at the beginning of the code with the other constants